fastlane/lib/fastlane/one_off.rb

```diff
@@ -17,14 +17,14 @@ def self.execute(args: nil)
         end
       end
 
-      raise "invalid syntax" unless action_name
+      UI.user_error!("invalid syntax") unless action_name
```

Comment:
I wonder if this should be `crash!` instead? seems like you might want a trace of how you got here?